nodejs/docfx.json

```diff
@@ -158,9 +158,7 @@
       "searchScope": [
         "Teams"
       ],
-      "feedback_system": "GitHub",
-      "feedback_github_repo": "MicrosoftDocs/msteams-docs",
```

Comment:
Let's retain `feedback_github_repo` for now; need to confirm w/Teams on feedback plan

Reply:
Will loop back here after base update.